cdc/owner/ddl_sink.go

```diff
@@ -143,11 +145,11 @@ func (s *ddlSinkImpl) run(ctx cdcContext.Context, id model.ChangeFeedID, info *m
 				if checkpointTs == 0 || checkpointTs <= lastCheckpointTs {
 					continue
 				}
-				lastCheckpointTs = checkpointTs
 				if err := s.sink.EmitCheckpointTs(ctx, checkpointTs); err != nil {
 					ctx.Throw(errors.Trace(err))
 					return
 				}
+				lastCheckpointTs = checkpointTs
```

Comment:
Please add comments about why adjusting the sequence of 
- `s.sink.EmitCheckpointTs`
- `lastCheckpointTs = checkpointTs`

Reply:
actually, we do not have to adjust the sequence, since `lastCheckpointTs` is a local variable which won't affect global execution.